ufobalapp/static/ufobalapp/js/tournament.js

```diff
@@ -209,7 +244,7 @@ app.controller("tournamentMatch", ["$scope", "$routeParams", "dataService", "$ti
                 prepareEvents(match);
                 $scope.match.halftimeLenght = $scope.match.tournament.halftime_length;
                 setTime(match);
-                match.team_one.color = "team-blue";
+                match.team_one.color = "team-blue"; // todo why team_one and team1
```

Comment:
moc nechápu TODO?

Reply:
jen jsem se te chtel zeptat proc se preliva team_one do team1

Reply:
hádám, že je to kvůli prohazování stran v klikátku